bin/faucet/.env

```diff
@@ -11,6 +11,7 @@ MIDEN_FAUCET_POW_CHALLENGE_LIFETIME=
 MIDEN_FAUCET_POW_CLEANUP_INTERVAL=
 MIDEN_FAUCET_POW_GROWTH_RATE=
 MIDEN_FAUCET_POW_BASELINE=
+MIDEN_FAUCET_POW_BASE_DIFFICULTY_AMOUNT=
```

Comment:
I wonder if the name should be just `MIDEN_FAUCET_BASE_AMOUNT`. I know it affects the PoW, but also, we could say that this is a more general setting for a "normal amount" that we expect to be requested.